### PR TITLE
* Some refining

### PR DESCRIPTION
**Pull request by Wagnerp**
* Some refinements

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/General/KryptonCommandType.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿#region BSD License
+/*
+ *
+ *   BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2023. All rights reserved.
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/General/KryptonCommandType.cs:5]: Incorrect start year

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

**Comment by Smurf-IV** [comment]
Will / Are These buttons accessible in the palette designer as well ?

**Comment by Wagnerp** [comment]
> Will / Are These buttons accessible in the palette designer as well ?

The ability to set custom images was done in the last PR, but I'll bring it back again.

File: Source/Krypton Components/Krypton.Toolkit/General/KryptonCommandType.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/General/KryptonCommandType.cs]: Corrected start year

**Pull request merged**